ods_ci/build/install_idp.sh

```diff
@@ -446,10 +430,35 @@ function validate_user_config_file(){
   validate_user_config_fields_and_values ldap
 }
 
-if [ "${USE_OCM_IDP}" -eq 1 ]
-      then
-          perform_ocm_login
-fi
+function update_test_config(){
+    echo "Update test config file..."
+
+    AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
```

Comment:
this would work only for the interop pipeline case, while this script is generic. You can add the feature if you want, but it'd be better to put it under an option of the script/env variable